CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ All notable changes to the Aptos TypeScript SDK will be captured in this file. T
 - Fix `deriveAccountFromPrivateKey` to use the "legacy" derivation path for Ed25519 keys first.
 - Deprecate `deriveAccountFromPrivateKey` as more inspection is needed from the user to determine the correct address.
 - Fix internal and example tests in regards to the change to zero state accounts
+- Remove `scriptComposer` api due to increase in the sdk bundle size
```

Comment:
Can we perhaps add instructions on how to use the script composer after this change?  Maybe add to the aptos.dev and then link to it